lldb/test/API/functionalities/watchpoint/modify-watchpoints/main.c

```diff
@@ -0,0 +1,17 @@
+#include <stdint.h>
+int main() {
+  int value = 5;
```

Comment:
If you take the address of "value" then it will always be in memory. Here we might be just getting lucky that the compiler choses to put this on the stack at -O0.